docs/deploying_apps/orgs_spaces_targets.md

```diff
@@ -25,4 +32,39 @@ Set the target with:
 
 Once you set the target, the Cloud Foundry client remembers it until you change it.
 
+You can change space without changing org using:
+
+``cf target -s SPACENAME``
+
+## Managing spaces
+
+You can create new spaces within an org if your account has the ``OrgManager`` role. To find out who has that role, run:
+
+``cf org-users ORGNAME``
+
+where ORGNAME is the name of the org. You will see a list of users and their roles.
+
+If you don't have the ``OrgManager`` role and you need to create a new space,  you should ask a co-worker who is an ``OrgManager`` for your organisation. If you need the ``OrgManager`` role to be added to your account, contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+As an ``OrgManager``, you can use:
+
+``cf create-space SPACENAME -o ORGNAME`` 
+
+to create a new space.
+
+You will then need to grant access to any tenant accounts who should be able to use that space. You do this by assigning a role. In most cases, the role you will assign is ``SpaceDeveloper``, which gives the account
```

Comment:
which gives the account.... ...ability to deploy and manage apps in the specified space?